runtime/src/iree/hal/utils/libmpi.c

```diff
@@ -67,21 +69,39 @@ iree_status_t iree_hal_mpi_library_load(
 // convert an MPI_Status result to an iree_status_t
 //
 // We convert MPI_SUCCESS to IREE_STATUS_SUCCESS, and MPI errors to
-// IREE_STATUS_INTERNAL and print the MPI error code.
-//
-// TODO(cascaval): use MPI_Error to print the actual error or consider
-// mapping to other error codes.
+// IREE_STATUS_INTERNAL and retrieve the error string from the MPI library.
 iree_status_t iree_hal_mpi_result_to_status(
     iree_hal_mpi_dynamic_symbols_t* syms, int result, const char* file,
     uint32_t line) {
-  iree_status_code_t code;
-  switch (result) {
-    case 0:  // MPI_SUCCESS
-      return iree_ok_status();
-    default:
-      code = IREE_STATUS_INTERNAL;
-      break;
+  // the MPI spec mandates MPI_SUCCESS == 0, all the other errors are
```

Comment:
```suggestion
  // The MPI spec mandates MPI_SUCCESS == 0, all the other errors are
```

https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar